helpers.lua

```diff
@@ -1269,3 +1269,11 @@ function courseplay:printMeThisTable(t,level,maxlevel,upperPath)
 		courseplay.alreadyPrinted = {};
 	end;
 end
+
+function courseplay:haveSameCourse(vehicleA, vehicleB)
+	if (vehicleA.cp.numWaypoints == vehicleB.cp.numWaypoints) then
+		return (vehicleA.waypoints == vehicleB.waypoints)
```

Comment:
Do we need to compare by content? 
It depends on how you assign the waypoint table to the vehicle, so I'd like your feedback in case I can leave comparison by reference or I need to write a deep compare.

Reply:
Regardless of this we should always use the Course class and never vehicle.waypoints (see my previous comment) and there is already a Course:equal() function.

Reply:
Switched to use previous equal check function.

Reply:
Reverted to fieldworkCourse:equals, as on original version.